tracer/src/Datadog.Trace/Telemetry/Metrics/MetricTags.cs

```diff
@@ -209,4 +209,16 @@ public enum InstrumentationError
         [Description("error_type:invoker")]Invoker,
         [Description("error_type:execution")]Execution,
     }
+
+    public enum WafAnalysis
+    {
+        // The generator splits on ; to add multiple tags
+        // Note the initial 'waf_version'. This is an optimisation to avoid multiple array allocations
+        // It is replaced with the "real" waf_version at runtime
+        [Description("waf_version;rule_triggered:false;request_blocked:false;waf_timeout:false;request_excluded:false")]Normal,
+        [Description("waf_version;rule_triggered:true;request_blocked:false;waf_timeout:false;request_excluded:false")]RuleTriggered,
+        [Description("waf_version;rule_triggered:true;request_blocked:true;waf_timeout:false;request_excluded:false")]RuleTriggeredAndBlocked,
+        [Description("waf_version;rule_triggered:false;request_blocked:false;waf_timeout:true;request_excluded:true")]WafTimeout,
```

Comment:
I made an error in the truth table.

```suggestion
        [Description("waf_version;rule_triggered:false;request_blocked:false;waf_timeout:true;request_excluded:false")]WafTimeout,
```